services/src/main/scala/cromwell/services/metadata/impl/MetadataServiceActor.scala

```diff
@@ -58,6 +64,14 @@ case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config, ser
   private val metadataReadRowNumberSafetyThreshold: Int =
     serviceConfig.getOrElse[Int]("metadata-read-row-number-safety-threshold", 1000000)
 
+  private val metadataTableMetricsInitialDelay = serviceConfig.getOrElse[FiniteDuration]("table-size-metrics-initial-delay", 5.minutes)
+  private val metadataTableMetricsInterval = serviceConfig.getOrElse[FiniteDuration]("table-size-metrics-interval", 1.hour)
+
+  private val metadataTableMetricsPath: NonEmptyList[String] = MetadataServiceActor.MetadataInstrumentationPrefix :+ "table"
+  private val dataFreeMetricsPath: NonEmptyList[String] = metadataTableMetricsPath :+ "data_free_in_gib"
+  private val dataLengthMetricsPath: NonEmptyList[String] = metadataTableMetricsPath :+ "data_length_in_gib"
+  private val indexLengthMetricsPath:  NonEmptyList[String] = metadataTableMetricsPath :+ "index_length_in_gib"
```

Comment:
As mentioned below, I think it's better to leave these metrics in bytes and let the graph side do the unit conversions.

```suggestion
  private val dataFreeMetricsPath: NonEmptyList[String] = metadataTableMetricsPath :+ "data_free"
  private val dataLengthMetricsPath: NonEmptyList[String] = metadataTableMetricsPath :+ "data_length"
  private val indexLengthMetricsPath:  NonEmptyList[String] = metadataTableMetricsPath :+ "index_length"
```